js/src/workspaces/window.js

```diff
@@ -594,19 +594,22 @@
         var dfd = jQuery.Deferred(),
         module = $.viewer.annotationEndpoint.module,
         options = $.viewer.annotationEndpoint.options;
-        if (_this.endpoint && _this.endpoint !== null) {
-          endpoint.set('dfd', dfd);
-          endpoint.search(_this.currentCanvasID);
-          // update with new search
+
```

Comment:
Sadly I created a branch from the master that contained the fix for the endpoint. This should be merged after that.